CONTRIBUTING.md

```diff
@@ -32,3 +32,27 @@ make develop DEV_ENV=/path/to/test/project/venv
 This will install an editable version of the package in the test project. Any
 changes made to the library will automatically reflect in the test project
 without the need to install the package again.
+
+## Releasing
+
+Assuming the latest version is 0.14.0 and you want to releaase 0.15.0,
+
+1. Run `make prepare-release VERSION=0.15.0` 
+   This will bump the version where neccesary, update the changelog and commit
+   all the changes to a new branch with name `release/v0.15.0`.
+2. Submit a Github PR for the new branch and merge it once it is approved.
+3. Switch to the main branch and pull the newly merged changes. Make sure you have the same commit
+   checked out that was auto-generated in step 1.
+4. Tag the commit with the version number
+   ```shell
+   git tag v0.15.0
+   ```
+5. Push the new tag to bot Github and GitLab
+   ```shell
+   git push github v0.15.0
+   git push gitlab v0.15.0
+   ```
+   This will kickoff a Gitlab release job which will publish packages to pypi.org
+   and create a new Github release.
```

Comment:
What/where is executed? I cannot find it in a few minutes 😉

Reply:
If you know, you know :)

Reply:
GitLab... I have read GitHub... Sorry 😄 
https://github.com/signalfx/splunk-otel-python/blob/main/.gitlab-ci.yml